examples/ruby/Gemfile

```diff
@@ -7,5 +7,5 @@ gem 'rake', '~> 13.0'
 gem 'rspec', '~> 3.0'
 gem 'rubocop', '~> 1.35'
 gem 'rubocop-rspec', '~> 3.0'
-gem 'selenium-devtools', '= 0.133.0'
-gem 'selenium-webdriver', '= 4.29.1'
+gem 'selenium-devtools', '= 0.134.0'
```

Comment:
Traditionally exact versions can be provided just by doing `'0.134.0'` (Same below)